serverless/knative_eventing/serverless-using-brokers.adoc

```diff
@@ -1,16 +1,18 @@
 [id="serverless-using-brokers"]
-= Using brokers with Knative Eventing
+= Using brokers as an event delivery mechanism
 include::modules/common-attributes.adoc[]
 include::modules/serverless-document-attributes.adoc[]
 :context: serverless-using-brokers
 
 toc::[]
 
 Knative Eventing uses the `default` broker unless otherwise specified.
+You can create this broker by using the `knative-eventing-injection` annotation.
 
-If you have cluster administrator permissions, you can create the `default` broker automatically using namespace annotation.
-
-All other users must create a broker using the manual process as described in this guide.
+[NOTE]
+====
+This annotation can be added by developers or cluster administrators, but can only be removed by a cluster administrator.
```

Comment:
Maybe:
> Although both developers and cluster administrators can add the `knative-eventing-injection` annotation to brokers, only cluster administrators can remove the annotation.

Reply:
It's not actually annotating the broker, it's annotating the namespace so that the broker gets created. I've updated the note so hopefully it's clearer.